app/src/renderer/vuex/modules/send.js

```diff
@@ -200,6 +200,11 @@ export default ({ node }) => {
       // check response code
       assertOk(res)
 
+      // Sometimes we get back failed transactions, which shows only by them having a `code` property
+      if (res.code) {
+        throw new Error(`Something went wrong while sending the transaction.`)
```

Comment:
any way we can be more specific?

Reply:
I agree we should display a better error, but we'd have to hardcode the base errors from the SDK. See https://github.com/cosmos/cosmos-sdk/issues/4013